multichain-aggregator/multichain-aggregator-entity/src/sea_orm_active_enums.rs

```diff
@@ -1,16 +1,17 @@
 //! `SeaORM` Entity, @generated by sea-orm-codegen 1.1.12
 
 use sea_orm::entity::prelude::*;
+use serde::{Deserialize, Serialize};
 
-#[derive(Debug, Clone, PartialEq, Eq, EnumIter, DeriveActiveEnum)]
+#[derive(Debug, Clone, PartialEq, Eq, EnumIter, DeriveActiveEnum, Serialize, Deserialize)]
 #[sea_orm(rs_type = "String", db_type = "Enum", enum_name = "hash_type")]
 pub enum HashType {
     #[sea_orm(string_value = "block")]
     Block,
     #[sea_orm(string_value = "transaction")]
     Transaction,
 }
-#[derive(Debug, Clone, PartialEq, Eq, EnumIter, DeriveActiveEnum)]
+#[derive(Debug, Clone, PartialEq, Eq, EnumIter, DeriveActiveEnum, Serialize, Deserialize)]
```

Comment:
did you added this manually? there is cli option to derive serde in sea-orm-cli: 
https://www.sea-ql.org/SeaORM/docs/generate-entity/sea-orm-cli/#generating-entity-files

Reply:
Yes, I added these derives by hand. I didn't like that `--with-serde` cli option adds serde impls for all created entities and there is no way to restrict it